trade.lic

```diff
@@ -0,0 +1,258 @@
+custom_require.call(%w(common common-items common-money common-travel drinfomon equipmanager))
+$debug = UserVars.trade_debug
```

Comment:
This is a global and so accessible outside of this script. I recommend a more unique name than `debug` (for example, `debug_mode_trade`. What do you think?

Reply:
Right, maybe it would make more sense just as a method in the class instead...
```ruby
def debugging?
  return UserVars.trade_debug
end
```

Reply:
I don't have a preference on that. My gut reaction is that won't make the code clearer, but it could...